app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ForgotPasswordsController < ApplicationController
+  def create
+    user = User.find_by_email(params[:email])
+    if user
+      user.generate_token
+      AppMailer.password_reset_link(user).deliver
+      render :confirm
```

Comment:
I would redirect here as well, always redirect from `POST`. You do not want the user to be posting here when he visits the same route unless there would be an error.